packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -174,7 +174,7 @@ export default class ElectronAppWrapper {
 					// so that it can tell us if we can really close the app or not.
 					// Search for "appClose" event for closing logic on renderer side.
 					event.preventDefault();
-					this.win_.webContents.send('appClose');
+					if (this.win_) this.win_.webContents.send('appClose');
```

Comment:
The code inside if (isGoingToExit) is running 3 times after closing the app and at the thirty time this.win_ is null. So, I think in some cases it's entering the next if statement if (!this.rendererProcessQuitReply_) and causing the problem. I hope it makes sense.